lib/internal/modules/esm/loader.js

```diff
@@ -56,6 +56,9 @@ function getTranslators() {
 }
 const { getOptionValue } = require('internal/options');
 
+const dc = require('diagnostics_channel');
+const onLoad = dc.tracingChannel('module.import');
```

Comment:
should we include `node` into the DC namespace (`node.module.import`)?

Reply:
We don't follow that pattern with any of our current events though. Probably fine as we've followed the `$module.$function` pattern so far, and we already disallow module collisions with userland. The `node:` prefix thing may mean we should have a corresponding `node.` prefix though.

I was thinking we could maybe create channel name aliases to allow for migration/deprecation paths for channel name changes. 🤔